src/main/java/net/minecraftforge/client/ForgeHooksClient.java

```diff
@@ -811,6 +814,7 @@ public static TextureAtlasSprite loadTextureAtlasSprite(
 
     private static final Map<ModelLayerLocation, Supplier<LayerDefinition>> layerDefinitions = new HashMap<>();
 
+    /** Call during {@link EntityRenderersEvent.RegisterLayerDefinitions} via the event method */
```

Comment:
Please deprecate this method instead and move it to the event so that it is impossible to call it outside the event.

Reply:
I ended up just removing all updates to ForgeHooksClient as the hook is entirely self contained in the event now, so I reverted the comment there. If preferred I can patch to deprecate it